.gitignore

```diff
@@ -12,3 +12,4 @@
 
 # Generated project artifacts
 **/dist
+__coverage__
```

Comment:
Do we actually generate code coverage? Also, can we not use the ugly Python-like `__something__` convention?

Reply:
This is the default configuration for Jest... not sure if you can change it 🤔

Reply:
We are generating code coverage, and we can change it. What would be the prefered way?